code/game/objects/effects/decals/Cleanable/misc.dm

```diff
@@ -206,3 +206,73 @@
 	icon_state = "asteroid0"
 	random_rotation = 2
 	random_icon_states = list("asteroid0", "asteroid1", "asteroid2", "asteroid3", "asteroid4", "asteroid5", "asteroid6","asteroid7","asteroid8")
+
+/obj/effect/decal/cleanable/graffiti_kot
```

Comment:
Perhaps create a parent graffiti object instead of repeating it several times?

Reply:
not sure it will works, since its parent is the cleanable obj i think? if i dont find a way to solve this, i could update it later when i will add more graffiti sprites, for now its only a few but yes, for later its better if i find a way to do it.

thanks for the review

Reply:
```
/obj/effect/decal/cleanable/graffiti
	name = "graffiti"
	desc = "Some sort of a graffiti."
	density = FALSE
	anchored = TRUE
	plane = -1
	icon = 'icons/effects/wall_graffiti.dmi'
	icon_state = "iriska"
	random_rotation = 0

/obj/effect/decal/cleanable/graffiti/security
	name = "graffiti"
	desc = "A vagabond beaten by IH, this is mercenary brutality."
	icon_state = "doodle"```

Do you understand what I mean by this?

Reply:
yep, dunno why my previous time it didnt worked, but now it works, i probably made typos on the path in spraypaint.dm

with this being done, that is all the issue told on github and discord solved.

thanks a lot for the advices.